.github/workflows/tangle.yml

```diff
@@ -0,0 +1,46 @@
+name: LMT Tangle
+
+on:
+  push:
+    branches: 
+      - master
+    paths:
```

Comment:
What is this for? The invocation below looks like it duplicates these file names. Assuming this is needed for something, could it be put in an environment variable that's used both here and in the command invocation?

Reply:
This list is used to tell GitHub which files to watch, and unfortunately cannot be put into an environment variable. As for using an anchor, that seems to not be implemented in GitHub’s YAML implementation. I’m considering reverting to checking all markdown files here, as it seems like it may be the best solution (even if it rebuilds on a few more files than needed)